ckb-contracts/tests/src/eth_recipient_typescript.rs

```diff
@@ -1,21 +1,22 @@
-use crate::utils::{case_builder::*, case_runner};
+// TODO capsule test do not support hash_type Byte(0x01) yet, so remove recipient tests temporally
```

Comment:
Is it possible to use code hash_type in test and dev, type hash_type in testnet and mainnet?

Reply:
Yes, i will use conditional compile just like SUDT script